docs/_posts/Damla-Gurbaz/2022-06-26-drug_brandname_ndc_mapper_en_3_0.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: model
+title: Mapping Drug Brand Names with Corresponding National Drug Codes
+author: John Snow Labs
+name: drug_brandname_ndc_mapper
+date: 2022-06-26
+tags: [chunk_mapper, ndc, clinical, licensed, en]
+task: Chunk Mapping
+language: en
+edition: Spark NLP for Healthcare 3.5.3
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained model maps drug brand names to corresponding National Drug Codes (NDC). Product NDCs for each strength are returned in result and metadata.
+
+## Predicted Entities
+
+`Strength_NDC`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/26.Chunk_Mapping.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/drug_brandname_ndc_mapper_en_3.5.3_3.0_1656260706121.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+
+```python
+document_assembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("chunk")
+
+chunkerMapper = ChunkMapperModel.pretrained("drug_brandname_ndc_mapper", "en", "clinical/models")\
+      .setInputCols(["chunk"])\
+      .setOutputCol("ndc")\
+      .setRels(["Strength_NDC"])\
+      .setLowerCase(True)
+
+
+pipeline = Pipeline().setStages([
+			document_assembler,
+			chunkerMapper])  
+
+
+model = pipeline.fit(spark.createDataFrame([[""]]).toDF("text")) 
+
+light_pipeline = LightPipeline(model)
+
+result = light_pipeline.fullAnnotate(["zytiga", "zyvana", "ZYVOX", "ZYTIGA"])
```

Comment:
Why do we use duplicate terms here like "zytiga"? I think we can remove one of them or use another term. Please don't forget to update Scala code and the results if you make any changes.